MAUI/ListView/filtering.md

```diff
@@ -11,6 +11,9 @@ documentation: ug
 
 This section explains how to filter the data and its related operations in the SfListView.
 
+To get start quickly with .NET MAUI ListView Filtering, you can check on this video:

```

Comment:
To get start quickly with filtering in .NET MAUI ListView,

Reply:
resolved